Corona-Warn-App/src/main/res/values-bg/strings.xml

```diff
@@ -602,11 +602,9 @@
     <!-- XTXT: onboarding privacy preserving analytics (ppa) - more info - other info point 1 text-->
     <string name="onboarding_ppa_more_info_other_info_point_1_text">"Посочената от Вас възрастова група"</string>
     <!-- XTXT: onboarding privacy preserving analytics (ppa) - more info - other info point 2 text-->
-    <string name="onboarding_ppa_more_info_other_info_point_2_text">"Посочената от Вас възрастова група"</string>
+    <string name="onboarding_ppa_more_info_other_info_point_2_text">"Времето между Вашето последно излагане с повишен риск и регистрирането на теста"</string>
```

Comment:
Can we please keep the string keys and don't swap them around, as we won't receive another translation delivery before the RC.